core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -808,4 +808,11 @@ void setSamplerConfiguration(String tableName, SamplerConfiguration samplerConfi
    * @since 1.8.0
    */
   SamplerConfiguration getSamplerConfiguration(String tableName) throws TableNotFoundException, AccumuloException, AccumuloSecurityException;
+
+  /**
+   * 
+   */
+  default SummaryRetriever summaries() {
```

Comment:
yeah.. I forgot to include the table name as a parameter, I intended to do that.